cli/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ _Released 7/16/2024 (PENDING)_
 **Bugfixes:**
 
 - Fixed an issue where the ReadStream used to upload a Test Replay recording could erroneously be re-used when retrying in cases of retryable upload failures. Fixes [#29227](https://github.com/cypress-io/cypress/issues/29227)
+- CLI output properly displays non-JSON response bodies when a Test Replay upload attempt returns a non-JSON response body for a non-200 status code. Addressed in [#29801](https://github.com/cypress-io/cypress/pull/29801).
```

Comment:
should this be marked as a bugfix or more of an improvement?

Reply:
I think it's a bug that we weren't appropriately handling non-JSON response bodies before this.

Reply:
That makes sense. My main concern is that the `verify-release-readiness` works correctly here which I believe only runs on the `develop` branch. Are you able to verify there are no changelog issues if you run the job locally?